### PR TITLE
Add Pie & Donut Chart Documentation

### PR DESCRIPTION
**Pull request by SyafiqMuizz**

### CODE REVIEW

File: visual/pie-donut-chart.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+![Retail Sales](./images/pie-donut-chart/output-2.PNG)
+
+>*NOTE:* Due to the size of the chart, some labels are cut off/extended outside the diagram. Instead, the legend will be used to assist in identifying the segments.
+
+![Store Type](./images/pie-donut-chart/output-2a.PNG)
+
+**Analysis**
+
+The conclusion of the use case chart displays the percentage of items bought from each product category and store type, where the Women's, juniors' & misses' wear takes up to almost half of the whole sales, with 47.12% across all store types. As the drill down was used for this use case, it is shown that Clothing & clothing accessories stores along with General merchandise stores contribute the most towards the sales of Women's, juniors' & misses' wear. From this, the management of the company should use the donut chart as an analysis tool to see which products should be focused on to garner in more revenue for the company, as well as which stores should specialise in selling said products to eventually gain a higher profit throughout their business year.
```
Comment (tchyong) [inline visual/pie-donut-chart.md]: > From this, the management of the company should use the donut chart as an analysis tool to see which products should be focused on to garner in more revenue for the company, as well as which stores should specialise in selling said products to eventually gain a higher profit throughout their business year.

i think is better to add below before your statement
From the distribution of the stores types, users are more tend to buy Women's, juniors' & misses' wear from specialist shop for clothing and general merchandise store (shopping mall like Parkson/Aeon/...) compare other as there are more choices.

**Comment by izairiabdlatif** [comment]
@SyafiqMuizz 

```A sample dataset is used to display the most common items sold compared to its store location. A donut chart will be used to display the data. Download sample data [here](./sample-data/pie-donut-chart/Retail Sales by Store Type.xlsx).```

Hyperlink to dataset is not shown as hyperlink

**Comment by izairiabdlatif** [comment]
@SyafiqMuizz 

```The Category field is used for the Category field, and the Sales This Category field is placed in the Value field```

This is kind of confusing. Can't the data column for Value just be called "Sales" instead of "Sales This Category"

File: visual/pie-donut-chart.md (hunk 1 above)
Comment (izairiabdlatif) [reply visual/pie-donut-chart.md]: @tchyong Changes implemented for this part

**Comment by izairiabdlatif** [comment]
> @SyafiqMuizz
> 
> `A sample dataset is used to display the most common items sold compared to its store location. A donut chart will be used to display the data. Download sample data [here](./sample-data/pie-donut-chart/Retail Sales by Store Type.xlsx).`
> 
> Hyperlink to dataset is not shown as hyperlink

@tchyong Changes implemented for this part

**Comment by izairiabdlatif** [comment]
> @SyafiqMuizz
> 
> `The Category field is used for the Category field, and the Sales This Category field is placed in the Value field`
> 
> This is kind of confusing. Can't the data column for Value just be called "Sales" instead of "Sales This Category"

@tchyong Changes implemented for this part

**Comment by tchyong** [comment]
# LGTM

**Pull request merged**